data/spells/lib/spells.lua

```diff
@@ -258,6 +258,16 @@ RANGE = {
 [16] = {1, 2},   [17] = {1, 2},   [18] = {1, 2}, [19] = {1, 2}
 }
 
+COORDINATES_AREA_CIRCLE5X5 = {
+{x = 0, y = 0},
+{x = 1, y = 0}, {x = 2, y = 0}, {x = 3, y = 0}, {x = 4, y = 0}, {x = 5, y = 0},
```

Comment:
I'll just leave the party condition function here and remove the others related to the bug fix (this table included), as it wouldn't take care of all the possible cases.